app/assets/stylesheets/settings.scss

```diff
@@ -1,4 +1,14 @@
-.ldap-form {
+.setting-form {
   margin: 0 auto;
   max-width: 400px;
 }
+
+select {
+  background-color: #FFF;
+  border: 1px solid #DDD;
```

Comment:
Color literals like `#DDD` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#DDD` should be written as `#ddd`